src/server/system_services/tier_server.js

```diff
@@ -538,11 +539,13 @@ function read_policy(req) {
     });
     pools = _.compact(pools);
     const pool_names = pools.map(pool => pool.name);
-    return P.join(
+    return Promise.all([
             nodes_client.instance().aggregate_nodes_by_pool(pool_names, req.system._id),
             nodes_client.instance().aggregate_hosts_by_pool(pool_names, req.system._id),
-        )
-        .spread(function(nodes_aggregate_pool, hosts_aggregate_pool) {
+        ])
+        .then(function(res) {
+            let nodes_aggregate_pool = res[0];
+            let hosts_aggregate_pool = res[1];
```

Comment:
```suggestion
        .then(([nodes_aggregate_pool, hosts_aggregate_pool]) {
```

Reply:
when doing that we get a lint error:
`Unexpected block statement surrounding arrow body; move the returned value immediately after the =>.eslint(arrow-body-style)`

Reply:
I was just missing the arrow `=>` Fixed the suggestion